docker-compose.relayers.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+version: '3'
+
+services:
+  relayer_1:
+    image: chainsafe/chainbridge-core-example:main
+    container_name: relayer_1
+    ports:
+      - "8545:8545"
```

Comment:
Why the need to expose ports?

Reply:
Now that you say it, actually we don't...